### ISSUE TITLE
Officials dont automatically gets editing rights

### ISSUE THREAD

**Issue by fellab** [UTNkar/moore#203]
### Description

For instance the officials of Forsränningen hasn't automatically gotten the right to add new roles and posts. 

### Steps to Reproduce

1. [First Step]
2. [Second Step]
3. [and so on...]

**Comment by Dekker1** [comment]
They actually can, you just didn't give the _forskå_ **group** access to the admin interface. Please make sure the following is toggled in their permissions:
![screen shot 2017-08-28 at 23 19 58](https://user-images.githubusercontent.com/1564252/29794277-a2cca2ea-8c47-11e7-8760-0e7591a1a5f2.png)

**Comment by fellab** [comment]
I thought that would automatically be received when they got the post? Wasn't.

**Comment by Dekker1** [comment]
For anyone in a post, they are automatically added and removed from the groups corresponding to the team. Officials are automatically given the right to govern the positions in their team.

This means that the amount of rights, is still mostly dependant on what rights are given to the group by administrators.

**Comment by fellab** [comment]
Not true. Fredrik Levin is vice project manager for the River Rafting and didn't automatically received moderating capability. Same for Mikaela Seleborg. Added them as "moderators" and "Forskå" and it worked.

**Comment by Dekker1** [comment]
That's definitely not where they should be. The group moderators can edit almost anything in the website. They should only be in the Forskå group and there roles should be "official" roles. If that's enabled they can access their recruitment mechanics

**Comment by fellab** [comment]
Where can I find "other permissions"?

**Comment by fellab** [comment]
I get it now. The problem I found though is that when someone is appointed to a position. I.e. Andrea Sjöberg is the project manager for Utnarm, her position role isn't automatically set as "Utnarm". That's why I manually had to add both Mikaela Seleborg and Fredrik Levin to Forskå, Erik Englund to Ledningsgruppen etc. 

![image](https://user-images.githubusercontent.com/29704138/30363364-211b2954-9860-11e7-9b3c-b7c7bb2ea54c.png)
